charts/ziti-router/values.yaml

```diff
@@ -82,6 +88,7 @@ tunnel:
   # -- built-in nameserver configuration, e.g. udp://127.1.2.3:53
   resolver: none
   # lanIf: eth0   # interface device name for tproxy?
+  # diverterPath: /path/to/userspace/bpf-program
```

Comment:
Rather than comment the value, it's better to leave it null with a documentation comment `# --` immediately above. That way, the value is documented in README by helm-docs, and tests like `{{ if .Values.diverterPath }}` will still evaluate false unless a non-null value is supplied.